tests/Unit/Console/Command/DebugCommandTest.php

```diff
@@ -37,4 +46,42 @@ public function testSuccessful(): void
         self::assertStringContainsString('profile.created', $content);
         self::assertStringContainsString(ProfileCreated::class, $content);
     }
+
+    public function testSuccessfulWithSubscribers(): void
+    {
+        $subscriber1 = new #[Subscriber('profile.test', RunMode::FromBeginning)]
+        class {
+        };
+
+        $subscriber2 = new #[Projector('profile.projection')]
+        class {
+            #[Subscribe(ProfileCreated::class)]
+            public function onProfileCreated(ProfileCreated $event): void
+            {
+            }
+        };
+
+        $command = new DebugCommand(
+            new AggregateRootRegistry(['profile' => Profile::class]),
+            new EventRegistry(['profile.created' => ProfileCreated::class]),
+            new MetadataSubscriberAccessorRepository([$subscriber1, $subscriber2]),
+        );
+
+        $input = new ArrayInput([]);
+        $output = new BufferedOutput();
+
+        $exitCode = $command->run($input, $output);
+
+        self::assertSame(0, $exitCode);
+
+        $content = $output->fetch();
+
+        self::assertStringContainsString('Subscribers', $content);
+        self::assertStringContainsString('events handled', $content);
+        self::assertStringContainsString('profile.test', $content);
+        self::assertStringContainsString('default', $content);
+        self::assertStringContainsString('profile.projection', $content);
+        self::assertStringContainsString('projector', $content);
+        self::assertStringContainsString('profile.created', $content);
```

Comment:
```suggestion
        self::assertStringContainsString('Subscribers', $content);
        self::assertStringContainsString('id', $content);
        self::assertStringContainsString('group', $content);
        self::assertStringContainsString('run mode', $content);
        self::assertStringContainsString('events handled', $content);
        
        self::assertStringContainsString('profile.test', $content);
        self::assertStringContainsString('default', $content);
        self::assertStringContainsString('from_beginning', $content);
        
        self::assertStringContainsString('profile.projection', $content);
        self::assertStringContainsString('projector', $content);
        self::assertStringContainsString('from_beginning', $content);
        self::assertStringContainsString('profile.created', $content);
```

This should remove one of the issue reported by infection, the mutation was removing the `id` column in the table.

Reply:
Resolved by commit https://github.com/patchlevel/event-sourcing/pull/721/commits/7d55f75d61611945756060e6622b4d75b6d1dec4